Tutorials/01-Cyton_Getting Started_Guide.md

```diff
@@ -452,28 +452,6 @@ You'll notice that the strongest alpha wave signals should be appearing in chann
 
 For more ideas on what to do next, check out the OpenBCI [Community Page] (http://openbci.com/community/) and the other OpenBCI Docs pages.
```

Comment:
I think the space after [Community Page] needs to be removed

Reply:
Everything else looks good. Thanks richard!

Reply:
This PR includes the general flow, readability, and functionality of the most used documents. There may be more drastic changes to the Ganglion docs that reflects the current state of the hardware, and also updated screenshots. Maybe better to contain that in another PR.